projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -67,6 +68,7 @@ class RobotFSM : FSM<State>::StateHandler
 {
     using Milliseconds = std::chrono::duration<double, std::milli>;
     using ImageDatabase = Navigation::ImageDatabase;
+    using AngleUnit = units::angle::degree_t;
```

Comment:
You're not actually using ``AngleUnit`` anywhere, right? So you don't need this line. This just means "make a new type called ``AngleUnit`` which is an alias for ``units::angle::degree_t``".